sql/core/src/test/scala/org/apache/spark/sql/execution/ui/AllExecutionsPageSuite.scala

```diff
@@ -32,10 +33,20 @@ import org.apache.spark.sql.test.SharedSQLContext
 import org.apache.spark.status.ElementTrackingStore
 import org.apache.spark.util.kvstore.InMemoryStore
 
-class AllExecutionsPageSuite extends SharedSQLContext {
+class AllExecutionsPageSuite extends SharedSQLContext with BeforeAndAfter {
 
   import testImplicits._
 
+  var kvstore: ElementTrackingStore = _
+
+  before {
+    kvstore = new ElementTrackingStore(new InMemoryStore, sparkContext.conf)
```

Comment:
Like in the other test, do you want to avoid creating it for every test, and only close it if it isn't null?

Reply:
As I have seen there is only two tests in this suite and both uses the kvstore.

Reply:
Yes. We can create the kvstore inside 'createStatusStore' method. In future, if any test cases doesn't required kvstore, we don't need create store. I will update.